enhancements/sig-architecture/26-policy-generator/README.md

```diff
@@ -0,0 +1,266 @@
+# Policy Generator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The goal of this feature is to allow automatic creation of OCM configuration policies, placement,
+and placement bindings files from Kubernetes resource object manifests (e.g. a ConfigMap YAML) and
+be able to deploy those policy files using existing GitOps mechanisms. This feature can lower the
+bar of policy configuration for customers.
+
+<!-- Based on these requirements, a Kustomize generator plugin will be created. This Kustomize plugin
+will be installed on the subscription controller so that the generator may be used directly through
+OCM GitOps. This means that a user will not need to store the generated policies in the Git
+repository, but only the Kustomize configuration. If the user so chooses, they may still store the
+generated policies in Git and have OCM apply the policies directly. -->
+
+## Motivation
+
+Users already have existing Kubernetes YAML files, so this will give users the ability to easily
+create tailored ACM policies from these YAML files rather than having to create individual policies
+manually from each file. Additionally, the policies can live in the same Git repo as the YAML,
+allowing for version control and simplicity.
+
+### Goals
+
+- Generic: No special handling is required. The configuration policy’s objectDefinition field will
+  contain exactly the same content as the input Kubernetes resource object manifest.
+- (Stretch Goal) Gatekeeper: In the case of a manifest with a gatekeeper policy, the automation must
```

Comment:
How about removing the stretch goals label on the ones we've already implemented?